mobify/progressive-web-scaffold
# Astro: Merlin's checkout closing alertView

Before closing the modal if the user is in checkout state (shipping/payment), displays an alert view confirming the user's action

 **JIRA**: https://mobify.atlassian.net/browse/HYB-985

## Changes
- created 2 events `checkout:alert-enabled` and `checkout:alert-disabled` that toggles cartModalHeader's flag to enable or disable the alert view

## How to test-drive this PR
- open the cart modal and close it, no alert view should appear
- open the cart modal and proceed to shipping or payment, alert view should appear
- open the cart modal and proceed to `done` state, alert view should not appear
